lib/typescript/httpclient/payload/ConnectChannelRequestPayload.ts

```diff
@@ -1,7 +1,6 @@
-export interface ConnectChannelRequestPayload {
-  source: string;
-  sourceChannelId: string;
-  token: string;
+export interface ConnectChannelFacebookRequestPayload {
+  pageId: string;
+  pageToken: string;
```

Comment:
I think now I understand a bit better why we're doing this (but the code  reads wrong anyway in the connect page as that's still a generic place).

We should rename the file  though